### DISCUSSION TITLE
Action for 'Certificates' keeps failing!

### DISCUSSION THREAD

**Discussion by Crystalflame66** [community/community#77788] [Other Feature Feedback, Questions, & Ideas]
*Revised*

**Comment by oct0f1sh** [comment]
Hey @bismahammid120389, you might want to post this one in the fastlane repo. Also, I see a basic auth header in your output for your certificates repo. If that's a sensitive auth token, I suggest you change it immediately as it is now leaked to all of GitHub.

**Reply by Crystalflame66** [reply]
Hey @oct0f1sh - Thanks so much for the reply and making me aware.
I have just deleted the thread as I didn't even realise! Thankyou for making me aware. I will repost as suggested ensuring any sensitive information is removed.

I hope nothing is showing now? I have updated the thread as *revised*.

**Reply by rogerluan** [reply]
@Crystalflame66 make sure you delete the history with your sensitive data, as it's still visible:

<img width="970" alt="image" src="https://github.com/community/community/assets/8419048/96ee51bc-9f3d-4803-9899-2d2f79f22414">

[example showing an edit history]

**Reply by Crystalflame66** [reply]
Hi @rogerluan - Thanks so much for your reply. I am unable to see anything sensitive showing that is relevant but still would really appreciate any guidance on how i can delete this thread. delete the history? I have never used this section of Github and not entirely sure of where i should be going. Any help would be massively appreciated! Thanks!

**Reply by rogerluan** [reply]
@Crystalflame66 your previous message includes an authorization header that might compromise your data (uncertain). You can delete the history of a previous message by clicking in the revision (in my screenshot above, it would be clicking the "Crystalflame66 created 20 hours ago" button) and then a pop up like this will appear:

<img width="674" alt="image" src="https://github.com/community/community/assets/8419048/880e7823-bf76-490e-a3c8-6335732ea203">

(It's different from yours because I can't see yours with the deletion button)

And then you press the delete revision.

**Reply by rogerluan** [reply]
I see you made it! 
<img width="531" alt="image" src="https://github.com/community/community/assets/8419048/209eb663-57ee-448b-919c-f5cc6798a974">

🎉

**Reply by Crystalflame66** [reply]
Hi @rogerluan - I literally just saw it as you responded. I have deleted the revision which i am hoping has removed the initial thread. Thank you so much for confirming and highlighting this! Can i just confirm this is no longer showing up for you? I think i am staying away from the community page moving forward... :D

**Reply by rogerluan** [reply]
Yup, it's no longer showing :)

This page is no different from any other, and what you just did and learned applies to any issue/pr/discussions in all of GitHub 😄 I definitely recommend sticking around and learning more about risks and how to use GitHub than staying away from it! There's a lot of value in the GitHub community ❤️ 

I hope you have a great week!

**Comment by Crystalflame66** [comment]
THANK YOU SO MUCH! I am banning myself from this LOL. Thank you so much for all your help guys! :)

**Comment by Crystalflame66** [comment]
Thank you so much for confirming!

I am always really frightened of things like this but lesson learnt! I'll be extra careful and aware for next time! 🤓

Definitely has been a learning experience... all this coding and building apps is an alien world to me! Feel like a deer in headlights 🤣

Thank you for being so supportive 🥰
A great community out there which is refreshing to see!

Have a wonderful week also! 😊



Sent from my Galaxy

-------- Original message --------
From: Roger Oba ***@***.***>
Date: 04/12/2023 13:37 (GMT+00:00)
To: community/community ***@***.***>
Cc: Crystalflame66 ***@***.***>, Mention ***@***.***>
Subject: Re: [community/community] Action for 'Certificates' keeps failing! (Discussion #77788)


Yup, it's no longer showing :)

This page is no different from any other, and what you just did and learned applies to any issue/pr/discussions in all of GitHub 😄 I definitely recommend sticking around and learning more about risks and how to use GitHub than staying away from it! There's a lot of value in the GitHub community ❤️

I hope you have a great week!
